src/services/Api.js

```diff
@@ -46,8 +47,7 @@ class Api {
       return response.data;
     } catch (error) {
       console.error(error.request);
-      toast.error(error.message);
-      toast.error(error.request.responseText);
+      toast.error(`${error.message}\n${error.request.statusText}`);
```

Comment:
error.request.statusText might not be available all the time, error.request can be null or undefined